docs/DeveloperGuide.adoc

```diff
@@ -827,15 +827,45 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
 [width="59%",cols="22%,<23%,<25%,<30%",options="header",]
 |=======================================================================
 |Priority |As a ... |I want to ... |So that I can...
-|`* * *` |new user |see usage instructions |refer to instructions when I forget how to use the App
 
-|`* * *` |user |add a new person |
+|`* * *` |user |add a new patient's particulars | know about their situation
 
-|`* * *` |user |delete a person |remove entries that I no longer need
+|`* * *` |user |edit my patient's particulars when the situation changes |
```

Comment:
Updating 'when the situation changes' might be more appropriate under 'So that I can...'